sos/policies/__init__.py

```diff
@@ -717,17 +717,8 @@ def _get_pkg_name_for_binary(self, binary):
     def get_cmd_for_compress_method(self, method, threads):
         cmd = method
         if cmd.startswith("xz"):
-            # XZ only determine number of threads to use (for 5.2 only) and
-            # set compression to -2
-            cmd = "%s -2" % cmd
-            try:
-                xz_package = self._get_pkg_name_for_binary(method)
-                xz_version = self.package_manager\
-                                 .all_pkgs()[xz_package]["version"]
-            except Exception as e:
-                xz_version = [u'0']  # deal like xz version is really old
-            if xz_version >= [u'5', u'2']:
-                cmd = "%s -T%d" % (cmd, threads)
```

Comment:
Are we 100% sure that `xz` older than 5.2 is no longer available in any of our active downstreams?

Reply:
I'm pretty confident - 5.2 was released in late 2014/early 2015 - https://github.com/xz-mirror/xz/releases
We're being less aggressive then with Python.

According to this https://pkgs.org/search/?q=xz CentOS 6 has 5.2 it too.  Debian (oldoldstable)/Ubuntu (16.04) are the last ones with old versions _ although in this case it's 5.1alpha.